app/script/conversation/EventMapper.coffee

```diff
@@ -130,6 +147,7 @@ class z.conversation.EventMapper
   _map_all_verified: ->
     message_et = new z.entity.Message()
```

Comment:
Yeah might be worth putting them together. Wanted to avoid to have all the cases in one message. But since verified is the only missing case we could totally do that. dont think e2ee message is the best option since its quite meaningless. you could argue that decryption errors should also be an e2e message. i would prefer sth like verification message.